quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java

```diff
@@ -51,4 +58,28 @@
         SignupUserResponse userResponse = new SignupUserResponse().id(createdUserEntity.getUuid()).status("USER SUCCESSFULLY REGISTERED");
         return new ResponseEntity<SignupUserResponse>(userResponse, HttpStatus.CREATED);
     }
-}
+
+    /**
+     * Method to signIn the application
+     * @param authorization
+     * @return
+     * @throws AuthenticationFailedException
+     */
+    @RequestMapping(method = RequestMethod.POST, path = "/user/signin", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<SigninResponse> login(@RequestHeader("authorization") final String authorization) throws AuthenticationFailedException {
+        byte[] decode = Base64.getDecoder().decode(authorization);
```

Comment:
we are require to use Basic authentication so "Basic username :Password" U didn't include "Basic" in ur authentication

Reply:
Changes made.